### ISSUE TITLE
How to test XDP_TX performance using Linux traffic gen?

### ISSUE THREAD

**Issue by williamtu** [microsoft/ebpf-for-windows#639]
Hi,

I have a sender Linux machine and receiver Windows machine, followed the xdp_test.exe guide
https://github.com/microsoft/ebpf-for-windows/blob/master/docs/GettingStarted.md#xdp_testsexe
to load the xdp program using netsh
```
netsh ebpf>add program reflect_packet.o xdp
netsh ebpf>show programs

    ID  Pins  Links  Mode       Type           Name
======  ====  =====  =========  =============  ====================
 65539     1      1  JIT        xdp            reflect_packet
```
And I have another Linux machine running DPDK test-me to generate traffic
```
root@t3600-2:~/dpdk/build/app# ./dpdk-testpmd -a 0000:02:00.0 -- -i --port-topology=chained --forward-mode=txonly --eth-peer=0,18:66:da:a2:62:6c --tx-ip=10.20.114.118,10.20.114.115
Waiting for lcores to finish...

  ---------------------- Forward statistics for port 0  ----------------------
  RX-packets: 0              RX-dropped: 0             RX-total: 0
  TX-packets: 512830304      TX-dropped: 748000        TX-total: 513578304
  ----------------------------------------------------------------------------
```
At Windows, I saw RX packets, but no TX packets.
<img width="1002" alt="Screen Shot 2021-10-18 at 6 37 58 AM" src="https://user-images.githubusercontent.com/5173499/137742822-f60f730e-0805-4f78-931f-3d9b7a52d365.png">

Question:
How do I know which Windows interface the XDP program binds to?
Is there a tool / command to know the XDP_TX packet rate?
(Or any pointer to the source code for me to read)

Thank you
William

**Comment by williamtu** [comment]
thank you!
I change my traffic gen to use the UDP port 8989, and unfortunately hit blue screen
"DRIVER IRQL NOT LESS THAN OR EQUAL"
![Screen Shot 2021-10-18 at 9 22 05 AM](https://user-images.githubusercontent.com/5173499/137778826-4db2a457-ee30-42a2-b0ce-6920827727eb.png)

How do I share the error info or debug this issue?
Thanks
William

**Comment by Alan-Jowett** [comment]
Interesting. @shankarseal can you take a look? I think something might be corrupted in the NBL as it's not finding the right send completion handler for the packet.

**Comment by williamtu** [comment]
that's great! thank you.

**Comment by dthaler** [comment]
Should be fixed by #641.  Please reopen if not.
